src/v/cloud_storage/remote.cc

```diff
@@ -789,11 +789,15 @@ ss::future<upload_result> remote::delete_objects(
             co_return upload_result::failed;
         }
 
+        // This is not ideal, we lose all non-failures but the first one, but
+        // returning a single result for multiple operations will lose
+        // information.
         co_return std::reduce(
           results.begin(),
           results.end(),
           upload_result::success,
           [](auto res_a, auto res_b) {
+              vlog(cst_log.trace, "Merging results {} and {}", res_a, res_b);
```

Comment:
It's awkward to be logging a deletion result without the context of which key we are deleting: if we remove this, can we just rely on the logging from the remote.cc code instead, to record the result of the operation at TRACE level?

Reply:
removed, I had added it for debugging